.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This function is for retrying commands in the case they fail. It accepts three arguments
+# $1: Number of retries it will attempt
+# $2: Command to execute
+# $3: Clean up commands
+execute_and_retry () {
+  retry_counter=0
+  max_retry=$1
+  while [ $retry_counter -lt $max_retry ]; do
+   attempt_failed=0
```

Comment:
Does this work with booleans?
```suggestion
   attempt_failed=false
```

Reply:
attempt_failed will record the exit response of the commands. By default that will be 0, but it could also be 1 or other values, so we want this to remain an integer